patches/wxWidgets/motion_notify.diff

```diff
@@ -0,0 +1,47 @@
+diff --git src/gtk/window.cpp src/gtk/window.cpp
+index 1f1c6244ec..0aef5f0987 100644
+--- src/gtk/window.cpp
++++ src/gtk/window.cpp
+@@ -1800,19 +1800,6 @@ gtk_window_motion_notify_callback( GtkWidget * WXUNUSED(widget),
+ 
+     wxCOMMON_CALLBACK_PROLOGUE(gdk_event, win);
+ 
+-    if (gdk_event->is_hint)
+-    {
+-        int x = 0;
+-        int y = 0;
+-#ifdef __WXGTK3__
+-        gdk_window_get_device_position(gdk_event->window, gdk_event->device, &x, &y, NULL);
+-#else
+-        gdk_window_get_pointer(gdk_event->window, &x, &y, NULL);
+-#endif
+-        gdk_event->x = x;
```

Comment:
Isn't the behavior now different? Here, the event to be processed gets the latest mouse coordinates. If you move that to the end of the function, where do the coordinates now come from?

Reply:
Right. My understanding is `gdk_event->x` and `y` are provided to the `gtk_window_motion_notify_callback` callback already filled in with the values from when the event happened. Calling `gdk_window_get_pointer` gets a more recent cursor position from the X server, so what the wx code was doing prior to my change was arguably wrong in that it was modifying an event to have newer data. 

I haven't tried printing the `gdk_event->x` though so this is speculation, but everything seems to work fine..